services/device-registry-mongodb/src/main/java/org/eclipse/hono/deviceregistry/mongodb/service/MongoDbBasedRegistrationService.java

```diff
@@ -227,7 +239,8 @@ public MongoDbBasedRegistrationService(
         Objects.requireNonNull(viaGroups);
         Objects.requireNonNull(span);
 
-        return processResolveGroupMembers(tenantId, viaGroups, span);
+        return tenantExists(tenantId, span)
```

Comment:
this is an internal operation that is invoked as part of one of the public API operations. The existence of the tenant should already have been checked there ...